x-pack/plugins/apm/public/components/alerting/ui_components/apm_rule_group_by.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiComboBox } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useCallback } from 'react';
+import { TRANSACTION_NAME } from '../../../../common/es_fields/apm';
+
+interface Props {
+  options: { groupBy: string[] | string | undefined };
+  preSelectedOptions: string[];
+  onChange: (groupBy: string | null | string[]) => void;
```

Comment:
tbh, this is a bit confusing 
`options: { groupBy: string[] | string | undefined };`
`onChange: (groupBy: string | null | string[]) => void;`

why the groupBy can be either a string and string[]?

Reply:
`string` should not be there. I forgot to remove it from here. I will change this.

Reply:
done